src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsToProceedApplicantNotificationHandler.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.notification;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.notify.NotificationService;
+import uk.gov.hmcts.reform.civil.notify.NotificationsProperties;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.NOTIFY_LIP_APPLICANT_CLAIMANT_CONFIRM_TO_PROCEED;
+import static uk.gov.hmcts.reform.civil.utils.PartyUtils.getPartyNameBasedOnType;
+
+@Service
+@RequiredArgsConstructor
+public class ClaimantResponseConfirmsToProceedApplicantNotificationHandler extends CallbackHandler
+    implements NotificationData {
+
+    private static final List<CaseEvent> EVENTS = List.of(NOTIFY_LIP_APPLICANT_CLAIMANT_CONFIRM_TO_PROCEED);
+
+    public static final String TASK_ID = "NotifyLiPApplicantClaimantConfirmToProceed";
+    private static final String REFERENCE_TEMPLATE = "claimant-confirms-to-proceed-applicant-notification-%s";
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
```

Comment:
Could you put this as a final field so that we dont have to create map every time we call the method?

Reply:
you mean like this?
![image](https://github.com/hmcts/civil-service/assets/107860057/7aba7e5e-fb67-4c6b-a533-346b40882e61)

Reply:
No i mean like this: 
`private final Map<String, Callback> callbackMap = Map.of(..your code for the map.);

protected Map<String, Callback> callbacks() {
   return callbackMap;
}
`

Reply:
Done, thanks!